estate/models/estate_property.py

```diff
@@ -0,0 +1,88 @@
+from odoo import models, fields, api
+from dateutil.relativedelta import relativedelta
+from odoo.exceptions import UserError
```

Comment:
```suggestion
from dateutil.relativedelta import relativedelta

from odoo import models, fields, api
from odoo.exceptions import UserError
```
First external libraries and then internal import.